docs/pages/web/toast.js

```diff
@@ -15,25 +16,39 @@ export default function DocsPage({ generatedDocGen }: {| generatedDocGen: DocGen
         name={generatedDocGen?.displayName}
         description={generatedDocGen?.description}
         defaultCode={`
-<Toast
-  primaryAction={{ accessibilityLabel: 'Test', label: 'Undo', size: 'lg' }}
-  text={
-    <Text inline> Saved to
-      <Link inline target="blank" href="https://www.pinterest.com/search/pins/?q=home%20decor">
-        Home decor
-      </Link>
-    </Text>
-  }
-  thumbnail={
-    <Image
-      alt="Modern ceramic vase pin."
-      naturalHeight={564}
-      naturalWidth={564}
-      src="https://i.ibb.co/Lx54BCT/stock1.jpg"
-    />
-  }
-/>
-        `}
+  <Toast
+    primaryAction={{ accessibilityLabel: 'Test', label: 'Undo', size: 'lg' }}
+    text={
+      <Text inline> Saved to
+        <Link inline target="blank" href="https://www.pinterest.com/search/pins/?q=home%20decor">
```

Comment:
Can we use `href="#"` so the user doesn't accidentally navigate away from the page?

Reply:
DONE